packages/modules/create-imodel/src/components/base-imodel/BaseIModel.tsx

```diff
@@ -273,54 +258,38 @@ export function BaseIModelPage(props: BaseIModelProps) {
       <div className="iac-imodel-base">
         <div className="iac-content-container">
           <Title>{updatedStrings.titleString}</Title>
-          <div className="iac-imodel-properties-container">
-            <div className="iac-inputs-container">
-              <LabeledInput
-                label={updatedStrings.nameString}
-                name="name"
-                setFocus
-                required
-                value={imodel.name}
-                onChange={onPropChange}
-                message={
-                  isPropertyInvalid(imodel.name)
-                    ? updatedStrings.nameTooLong
-                    : undefined
-                }
-                status={isPropertyInvalid(imodel.name) ? "negative" : undefined}
-                autoComplete="off"
-              />
-              <LabeledTextarea
-                label={updatedStrings.descriptionString ?? "Description"}
-                name="description"
-                value={imodel.description}
-                onChange={onPropChange}
-                rows={4}
-                message={
-                  isPropertyInvalid(imodel.description)
-                    ? updatedStrings.descriptionTooLong
-                    : undefined
-                }
-                status={
-                  isPropertyInvalid(imodel.description) ? "negative" : undefined
-                }
-                autoComplete="off"
-              />
-              {!extentComponent && (
+          {props?.children ?? (
```

Comment:
hmm.. whole layout will also go to the user now as we have few classes here inside...
wondering if we should still keep some of it :/ not sure yet

Reply:
as discussed we can customize package by passing some class as props, this can be handled in separate PBI